scaladoc/src/dotty/tools/scaladoc/site/StaticSiteContext.scala

```diff
@@ -19,17 +19,21 @@ class StaticSiteContext(
 
   var memberLinkResolver: String => Option[DRI] = _ => None
 
-  def indexTemplate(): LoadedTemplate =
+  private def indexFiles =
     val files = List(new File(root, "index.html"), new File(root, "index.md")).filter { _.exists() }
 
     if files.size > 1 then
       val msg = s"ERROR: Multiple root index pages found: ${files.map(_.getAbsolutePath)}"
       report.error(msg)
+    files
```

Comment:
It should be `indexFile` as we get only one file (either html or md) but not both at once. Moreover it would never be files becuase of assertion in line 25